### PR TITLE
Css two column layout. -Janice Lee

### PR DESCRIPTION
**Pull request by janiicelee**
Screen Shot 2022-01-28 at 5 39 28 PM

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    <div class="row">
+      <div class="header">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Iste omnis maxime iure quia cum, nisi dolore molestias aspernatur earum reiciendis neque
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:23]: How big is the column? Your `column` class tells me nothing about its purpose.

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="header">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:15]: Isn't this a `column` of some sort? You should use a `column` class to properly set up your layout class structure.

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  font-family: Karla, sans-serif;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column {
+  display: flex;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:33]: If you find yourself trying to apply `display: flex` to a `column` class element, that really means you need a `row` class element **_inside_** that `column` class element. Your two divs in that `column` (one for the image and one for the text) should be their own `column` class elements inside that `row` within this column.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by janiicelee** [comment]
Screen Shot 2022-01-31 at 1 04 03 PM

File: css-two-column-layout/styles.css [hunk 4]
```diff
@@ -49,8 +47,10 @@ button {
 }
 
 img {
+  float: left;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:50]: Try doing this without using `float`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by janiicelee** [comment]
Screen Shot 2022-01-31 at 4 57 51 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**